src/olympia/reviewers/serializers.py

```diff
@@ -586,3 +580,16 @@ def validate(self, data):
                     'You can\'t submit a line number without associating '
                     'it to a filename.')})
         return data
+
+
+class DraftCommentSerializerReadOnly(DraftCommentSerializer):
+    version_id = serializers.SerializerMethodField()
+
+    class Meta(DraftCommentSerializer.Meta):
+        fields = (
+            'id', 'filename', 'lineno', 'comment',
+            'version_id', 'user', 'canned_response'
+        )
+
+    def get_version_id(self, obj):
+        return obj.version.id
```

Comment:
So thinking about the parent `DraftCommentSerializer` class - it's got a `SplitField` which would SET a version (a pk) and GET a full serialized response - but when would that full response ever be used if the get responses are handled by `DraftCommentSerializerReadOnly`?

Reply:
Is the solution just to change `DraftCommentSerializer` instead to always be a version_id?

Reply:
That was the very first thing I tried, and it seemed fine for the `list` and `retrieve` cases, but broke all the tests for `create`. I, possibly incorrectly, assumed that those cases needed access to the `version` object, instead of `version_id`, which led me to this solution. I was just trying to get a result that works, but I think I went the wrong direction. How would I tell a single serializer that I need to be able to set a `version` (which is really just a `pk`), but that I don't want to return a `version` object, and rather only want to return a `version_id`?

Reply:
@eviljeff I guess I should have flagged you or asked for another review, as I think you might have missed the fact that I asked a question here. Can you help?